src/App.tsx

```diff
@@ -14,11 +14,81 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGood: string[];
+};
+
+class App extends React.Component <{}, State> {
+  state = {
+    selectedGood: [goodsFromServer[8]],
```

Comment:
```suggestion
    selectedGood: [],
```
you can left like this in initial state